Rakefile

```diff
@@ -43,6 +43,27 @@ $doEnableLuaGcProfile= $buildTarget == "Release" ? 0 : 1
 $doBuildAdmob=0
 $doBuildFacebook=0
 
+# Relative path of the telemetry client files in the SDK 
+$telemetryClient = "telemetry/www/"
```

Comment:
This is relative to `pkg/sdk/`. I'm not sure what the `sdk/` dir is, but as far as I can see, it's used for common files when building various different platforms. I only copy telemetry on packaging, so I used the `pkg` as the other packaging stuff does for preparation.